OpenRA.Game/Scripting/ScriptMemberWrapper.cs

```diff
@@ -121,7 +121,7 @@ public void Set(LuaRuntime runtime, LuaValue value)
 				throw new LuaException("The property '{0}' is read-only".F(Member.Name));
 		}
 
-		public static IEnumerable<MemberInfo> WrappableMembers(Type t)
+		public static IEnumerable<MemberInfo> WrappableMembers(Type t, bool onlyAISafeMethods)
```

Comment:
This feels like the wrong place to be doing this.  Please remind me to come back to this if I don't by myself.